src/Wishlist.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+import ListItem from './ListItem.js';
+
+export default function Wishlist({ listItems }) {
+  return (
+    <Wrapper>
+      {listItems.length > 0 ? (
+        listItems.map((listItem, index) => (
+          <ListItem
+            key={index}
+            destination={listItem.destination}
+            notes={listItem.notes}
+          />
+        ))
+      ) : (
+        <Message>
+          You currently have nothing on your wishlist. Start by adding some
+          destinations you would like to dive.
+        </Message>
+      )}
+    </Wrapper>
```

Comment:
Since your `<ListItem>`is a section element as well, you are rendering sections inside a section. Looks like this is allowed, but it seems strange to me :D

Reply:
Yeah, I actually checked that as I also thought it's a bit strange.